types/node/child_process.d.ts

```diff
@@ -7,7 +7,7 @@ declare module "child_process" {
         stdin: Writable | null;
         stdout: Readable | null;
         stderr: Readable | null;
-        readonly channel?: Pipe | null;
+        readonly channel?: Pipe;
```

Comment:
I tried following in repl:
```
> let a = cp.spawn("dir", { stdio: [null, null, null, 'ipc'] })
undefined
> a.channel
null
```
So it seems `null` happens.

Reply:
@Flarna So it is the Node.js's docs that is insufficient. I'm closing this.

Reply:
Yes, seems so. `null` and `undefined` are often intermixed as people tend to use `== null` or `== undefined` checks which end up in the same.